content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,67 @@
+- **Added** support for configuring the secrete paths for Istio mutual TLS certificates.
+- **Added** support for [PKCS 8](https://en.wikipedia.org/wiki/PKCS_8) private keys for workloads, enabled by the field `pkcs8-keys` on Citadel.
+- **Added** jitter(need a link) when sending CSR requests for certification generation.
+- **Improved** JWT public key fetching logic to be more resilient to network failure.
+- **Fixed** SAN field in workload certificates is set as `critical`. - need clear explanation
+- **Fixed** mutual TLS probe rewrite for HTTPS probes.
+
+## Telemetry
+
+- - **Added** Full support for control over Envoy stats generation, based on stats prefixes, suffixes, and regular expressions through the use of [annotations](https://preliminary.istio.io/docs/reference/commands/pilot-agent/#annotations).
+- **Changed** Prometheus generated traffic is excluded from metrics.
+- **Added** support for sending traces to Datadog.
+
+## Policy
+
+- **Fixed** TCP Policy enforcement in Mixer.
+- Any channge in istio RBAC?
+
+## Configuration Management
+
+- **Enabled** webhook validation of more CRDs.
+
+## Installation and Upgrade
+
+- **Updated** default proxy memory limit size from XX to XX.  Explain why?
+- **Added** pod anti-affinity and toleration support to all of our control plane components. (link to k8s docs)
+- **Added** `sidecarInjectorWebhook.neverInjectSelector` and `sidecarInjectorWebhook.alwaysInjectSelector` to allow users to further refine whether workloads should have sidecar automatically injected or not, based on label selectors.
+- **Added** `global.logging.level` and `global.proxy.logLevel` to allow users to easily configure logs for control plane and data plane components globally.
+- **Added** support to configure the Datadog location via `global.tracer.datadog.address`. (link to the page)
+- **Removed** [deprecated Adapter and Template CRDs](https://discuss.istio.io/t/deprecation-notice-custom-mixer-adapter-crds/2055). Configure `mixer.templates.useTemplateCRDs=true` and `mixer.adapters.useAdapterCRDs=true` to re-enable them.
+
+Refer to the [installation option change page](/docs/reference/config/installation-options-changes/) to view the complete list of changes.
+
+## `istioctl` and `kubectl`
+
+- **Graduated** `istioctl verify-install` out of experimental.
+- **Improved** `istioctl verify-install` to validate if a given Kubernetes environment meets Istio's prerequisites.
+- **Added** auto-completion support to `istioctl`.
+- **Added** `istioctl experimental dashboard` to allow users to easily open the web UI of any Istio addons.
+- **Added** `istioctl x` alias to conveniently run `istioctl experimental` command.
+- **Improved** `istioctl version` to report both Istio control plane and `istioctl` version info by default.
+- **Improved** `istioctl validate` to validate Mixer configuration and supports deep validation with referential integrity.
+
+## Other
+
```

Comment:
Add an item about Istio CNI:
```
- **Added** Istio CNI support to setup sidecar network redirection and remove the use of `istio-init` containers requiring `NET_ADMIN` capability.
```

I don't see any notes about limitations / known issues in this doc, but I'd document one known issue about Istio CNI in 1.2:
```
Known issue: Istio CNI doesn't honor `global.proxy.*` redirection settings.
```

Reply:
@louiscryan prefers to leave the known issue out of this doc.

Reply:
@rlenglet can I use improved instead? (since this is in 1.1)  It would be good to highlight what has been improved.

Reply:
@linsun It seems like it was not included in any release notes for 1.1.x releases, which is why I thought it'd be appropriate to word it this way.

Reply:
WOW, I know it is been in 1.1 docs... perhaps not highlighted in relnnote

Reply:
added